weave/trace/op.py

```diff
@@ -417,15 +417,19 @@ def _do_call(
     # Handle all of the possible cases where we would skip tracing.
     if settings.should_disable_weave():
         res = func(*pargs.args, **pargs.kwargs)
+        call.output = res
```

Comment:
Why do we have this?  Is weave is disabled, would the call object get tracked anyways?

Reply:
this allows the non-tracked `call` to have the correct output for downstream users

Reply:
When would a downstream user need the call if weave is disabled?  Can you give me an example of how that would be used?

Reply:
Well let's say they are using an API like:

```
res, call = my_op.call(...)
if call.output > 0.5:
    do_a()
else
    do_b()
```

This code path is valid when tracing is on, but brakes when you disable it.

Reply:
hmm, ok I see the point here.

This raises a separate concern about whether users can rely on the shape of `call` even when weave is disabled, but I won't block here